### DISCUSSION TITLE
I'm search web3 dev

### DISCUSSION THREAD

**Discussion by olivereth** [ethereum/ethereum-org-website#13348] [General]
Greetings everyone, our team is looking for employees. We need Web3 developers, Nft artists, beta testers, managers, advertisers. We are ready to pay from 10.000$ per month. All you need is a computer, a couple free hours a day and a desire to work. Also we have good bonuses, career growth, friendly team. Our company has a certificate, before starting work is made prepayment. If you are interested, then write to me in private messages.
My contacts:
telegram: @web3oliver
discord: ethereum033

**Comment by corwintines** [comment]
Not appropriate for here.
